components/dialpad/package.json

```
@@ -3,6 +3,7 @@
   "version": "0.0.1",
```

We should update the version when the package.json file is updated.
```suggestion
  "version": "0.0.2",
```